core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2700,10 +2868,20 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
             path.toString(), options.getOwner(), options.getGroup(), e.toString()), e);
       }
     }
+    String commandName = rootRequired ? "chown" :
```

Comment:
check `options.getOwner() != null` instead of `rootRequired` - it's confusing that root being required implies that the name of the operation should be "chown"